openfisca_france/model/prelevements_obligatoires/impot_revenu/charges_deductibles.py

```diff
@@ -469,26 +443,23 @@ def function(self, simulation, period):
         return period, cd1 + cd2
 
 
-class cd_penali(Variable):
-    column = FloatCol(default = 0)
+class pensions_alimentaires_deduites(Variable):
```

Comment:
Note that it's still "in progress" (see the label ^^) so I'm sure @benjello will cope with the release stuff when finished.